config/helm/aws-node-termination-handler/values.yaml

```diff
@@ -76,6 +76,9 @@ taintNode: false
 # Log messages in JSON format.
 jsonLogging: false
 
+# Sets the log level
+logLevel: "info"
```

Comment:
should we leave this as `""` ?

Reply:
My thought is it's easier to figure out the default value this way.

 If we leave it as `""` , then helm readme would have default value say `""` too (even though app will default to 'info') OR we have the helm default value actually say `info`, but then the value in values.yaml would be `""` and not match which is confusing/doesn't align with how we documented other vals.